irohad/torii/processor/impl/query_processor_impl.cpp

```diff
@@ -27,21 +27,18 @@ namespace iroha {
         : qpf_(qpf), validator_(stateless_validator) {}
 
     void QueryProcessorImpl::query_handle(const model::Query &query) {
```

Comment:
Here references are bad solution because because life time of query inside QueryProcessor may be greater than  outside code. This issue may be solved sending by value of by smart pointer.